docs/source/installation/guide_complete/index.rst

```diff
@@ -0,0 +1,363 @@
+| Parameters   | not available             |  n.a.         | These are additional broker parameters that are typically encoded as URL parameters, for example, to specify SSL        |
+|              |                           |               | parameters such as the filepath to the certificate that is to be used. The parameters are currently not definable       |
+|              |                           |               | through the CLI but have to be added manually in the ``config.json``. A key ``broker_parameters`` should be added that  |
+|              |                           |               | is a dictionary, which can contain fields: ``cafile``, ``capath``, ``cadata``, ``certfile``, ``keyfile`` and            |
+|              |                           |               | ``no_verify_ssl``.                                                                                                      |
++--------------+---------------------------+---------------+-------------------------------------------------------------------------------------------------------------------------+
+
+
+.. _installation:guide-complete:create-profile:core-sqlite-dos:
+
+``core.sqlite_dos``
+-------------------
+
+This storage plugin uses `SQLite <https://sqlite.org/>`_ and the `disk-objectstore <https://disk-objectstore.readthedocs.io/en/latest/>`_ to store data.
+The ``disk-objectstore`` is a Python package that is automatically installed as a dependency when installing ``aiida-core``, which was covered in the :ref:`Python package installation section <installation:guide-complete:python-package>`.
+The installation instructions for SQLite depend on your system; please visit the `SQLite website <https://www.sqlite.org/download.html>`_ for details.
+
+Once the prerequisistes are met, create a profile with:
+
+.. code-block:: console
+
+    verdi profile setup core.sqlite_dos
+
+The options specific to the ``core.sqlite_dos`` storage plugin are:
+
+* ``--filepath``: Filepath of the directory in which to store data for this backend.
+
+
+.. _installation:guide-complete:create-profile:core-psql-dos:
+
+``core.psql_dos``
+-----------------
+
+This storage plugin uses `PostgreSQL <https://www.postgresql.org/>`_ and the `disk-objectstore <https://disk-objectstore.readthedocs.io/en/latest/>`_ to store data.
+The ``disk-objectstore`` is a Python package that is automatically installed as a dependency when installing ``aiida-core``, which was covered in the :ref:`Python package installation section <installation:guide-complete:python-package>`.
+The storage plugin can connect to a PostgreSQL instance running on the localhost or on a server that can be reached over the internet.
+Instructions for installing PostgreSQL is beyond the scope of this guide.
+
+Before creating a profile, a database (and optionally a custom database user) has to be created.
```

Comment:
EDIT: Just realized that `verdi presto --use-postgres` basically does that. So for me, it's fine to only keep a manual version here, as `verdi presto --use-postgres --profile-name <profile-name>` with RMQ installed allows for a fully performant custom named profile. This is something we should also mention somewhere. Right now, it reads as if `verdi presto` can only be used for the less performant profile. Personally, I see myself using the command in that way as a shortcut, as I don't want to have to manually do the PostgreSQL steps every time I create a `core.psql_dos` profile.

With the deprecation of `verdi quicksetup` in favor of `verdi profile setup`, there is now no option that AiiDA automatically takes care of the PostgreSQL steps? I think this would still be a very nice feature to have here (e.g. via a `--auto-postgres` flag), so that users don't have to deal with `psql` and `CREATE USER` and `CREATE DATABASE` themselves, which might present a hurdle.